src/coffee/client.coffee

```diff
@@ -220,9 +220,8 @@ class SphereClient
   #
   # options - An {Object} to configure the client
   constructor: (options = {}) ->
-
     # Private: instance of a {TaskQueue}
-    @_task = options.task or new TaskQueue
+    @_task = options.task or new RepeaterTaskQueue {}, { attempts: 50, timeout: 200, timeoutType: 'v' }
```

Comment:
Prefer letting them customise the number so there will be one option/flag: to modify the amount of retries and to disable it completely. So setting that option to 0 would disable it.